README.md

```diff
@@ -26,24 +26,24 @@ puts doc
 #### I made a mistake in the RubyExam, can you find it?
 
 ```ruby
-module RubyExam
+class RubyExam
   def initialize
     @score = 0
     @current_question = 1
   end
 
   def got_one_right
-    @score++
+    @score += 1
     next
```

Comment:
`redo` and `next` are keywords in ruby so we should really use something else. The methods will still work if you call them on the instance (`exam.next`/`self.next`).